### PR TITLE
Remove cqtx log field

### LINKED ISSUE #10066: %<cqtx> log field shows HTTP/1.1 for HTTP/2 requests
### Issue 
The `%<cqtx>` logs the client request line consisting of themethod, uri and the http version. The http version is inaccurate for http/2 requests:
```
127.0.0.1 - - [18/Jul/2023:23:01:45 -0000] "POST https://127.0.0.1:61016/some/path3?foo=bar HTTP/1.1" 404 0
```

### workaround
To capture the client request line with the accurate http version, the  `%<cqtx>` field can be replaced by `%<cqhm> %<cqu> %<cqpv>` , like in the following `logging.yaml`:
```
logging:
  formats:
    - name: old
      format: '%<chi> - %<caun> [%<cqtn>] "%<cqtx>" %<pssc> %<pscl>'
    - name: new
      format: '%<chi> - %<caun> [%<cqtn>] "%<cqhm> %<cqu> %<cqpv>" %<pssc> %<pscl>'
  logs:
    - filename: access
      format: old
    - filename: access-new
      format: new
```

### PR DESCRIPTION
**Pull request by lzx404243**
This fixes #10066.

### CODE REVIEW

File: doc/release-notes/upgrading.en.rst [hunk 1]
```diff
@@ -85,3 +85,8 @@ Metrics
 ------------------
 
 The HTTP connection metric proxy.process.http.dead_server.no_requests has been renamed to proxy.process.http.down_server.no_requests.
+
+Logging
+------------------
+
+The ``cqtx`` log field has been deprecated.
```
Comment (serrislew) [inline doc/release-notes/upgrading.en.rst:92]: "deprecated but can still be replaced by ``cqhm pqu cqpv``"

File: doc/admin-guide/logging/formatting.en.rst [hunk 2]
```diff
@@ -584,19 +584,18 @@ sshv  Origin Response       Origin server's response HTTP version.
 Request Details
 ~~~~~~~~~~~~~~~
 
-.. _cqtx:
-
-The following logging fields are used to obtain the actual HTTP request
-details.
+The following logging fields show the details of the actual HTTP request and can be used in combination to reproduce a HTTP/1.1 request line (e.g. ``GET http://www.company.com HTTP/1.0``).
```
[the remark this answers is not in the document]
Comment (lzx404243) [reply doc/admin-guide/logging/formatting.en.rst]: Good point. Updated that

File: doc/release-notes/upgrading.en.rst (hunk 1 above)
Comment (lzx404243) [reply doc/release-notes/upgrading.en.rst]: Yeah it's good to mention the replacement options. Added that.

**Comment by bneradt** [comment]
[approve ci]

**Review by serrislew — DISMISSED** [review]

**Review by serrislew — APPROVED** [review]

**Pull request merged**